torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -321,8 +330,8 @@ def rotate_image_tensor(
     angle: float,
     interpolation: InterpolationMode = InterpolationMode.NEAREST,
     expand: bool = False,
-    fill: Optional[List[float]] = None,
     center: Optional[List[float]] = None,
+    fill: Optional[List[float]] = None,
```

Comment:
On the same file, fill is always before center. Why change it here?

Reply:
I see, thanks for pointing out. I was trying to align to v1 `F.rotate` where it is `center` -> `fill`. But, looks like `F.affine` has `fill` -> `center`...